tests/test_repository.py

```diff
@@ -112,37 +53,19 @@ def f(pkg_name: str) -> bool:
 @pytest.mark.skipif(
     OS_VERSION == "tumbleweed", reason="No testing for openSUSE"
 )
-@pytest.mark.parametrize(
-    "container_per_test", [REPOCLOSURE_CONTAINER], indirect=True
-)
-def test_repoclosure(container_per_test):
-    """Run :command:`dnf repoclosure` on all packages in the ``SLE_BCI`` repository
-    excluding the packages in :py:const`REPOCLOSURE_FALSE_POSITIVES`.
-
-    """
-    package_list = list(
-        filter(
-            package_name_filter_func(
-                REPOCLOSURE_FALSE_POSITIVES + KNOWN_BROKEN
-            ),
-            get_package_list(container_per_test.connection),
-        )
+@pytest.mark.parametrize("container_per_test", [BASE_CONTAINER], indirect=True)
+def test_installcheck(container_per_test):
+    """Run installcheck against the SLE_BCI repo + locally installed packages."""
+    container_per_test.connection.run_expect(
+        [0],
+        "zypper ref && installcheck $(uname -m) --exclude 'rpm-ndb' /var/cache/zypp/solv/{SLE_BCI,@System}/solv",
```

Comment:
why are we checking `@System/solv` ? the packages wouldn't be installed if they fail to install. if its just to fulfill dependencies, then add it after a `--nocheck` parameter. 

but imho it could be all left out alltogether. 

also, to be debated, it should be run `--withobsoletes` which will throw a bunch of errors that we imho should be cleaning up

Reply:
> why are we checking @System/solv ? the packages wouldn't be installed if they fail to install. if its just to fulfill dependencies, then add it after a --nocheck parameter.

Can do.

> but imho it could be all left out alltogether.

No, then everything fails because of aaa_base -> distribution-release.

> also, to be debated, it should be run --withobsoletes which will throw a bunch of errors that we imho should be cleaning up

IMO for a later step

Reply:
ok, then we should add it after --nocheck, something like

```
zypper ref && cd /var/cache/zypp/solv && installcheck $(uname -m) --exclude .. SLE_BCI.solv --nochecks System.solv```

Reply:
Done